userguide/one_shot_object_detection/README.md

```diff
@@ -62,18 +61,59 @@ To learn more about deploying One-Shot to CoreML details, refer to the [Object D
 
 ## How it works
 
-In this section, we will go into detail on what happens under the hood of the `create` method. An OSOD model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train an object detector.
+In this section, we will go into detail on what happens under the hood of the `create` method. A One-Shot Object Detector model is trained in two steps: First, we use the starter images to generate synthetic data and annotations, and second, we train a model on the generated synthetic data.
 
 **Stage 1: Automatic synthetic data generation**
 
-To begin training, a set of synthetic training data needs to be generated using the provided starter image(s). 
+To begin training, the provided starter image(s) are used to generate a set of synthetic training data. 
 The synthetic images are generated by applying yaw, pitch, and roll rotations on a set of background images. If no background images are provided, a set of default background images are automatically downloaded and used.
 
 **Stage 2: Train the Object Detector model**
 
 To learn about the training implementation details, refer to the [Object Detector: How it Works](https://apple.github.io/turicreate/docs/userguide/object_detection/how-it-works.html) chapter.
 
+## Troubleshooting
```

Comment:
It's usually not good to have a section and start with the subsection immediately.